frontend/src/pages/modelRegistry/ModelRegistryCoreLoader.tsx

```diff
@@ -1,10 +1,11 @@
 import * as React from 'react';
 import { Outlet } from 'react-router';
-import { MODEL_REGISTRY_DEFAULT_NAMESPACE } from '~/concepts/modelRegistry/const';
+
 import { ModelRegistryContextProvider } from '~/concepts/modelRegistry/context/ModelRegistryContext';
 
+// TODO: Parametrize this to make the route dynamic
 const ModelRegistryCoreLoader: React.FC = () => (
-  <ModelRegistryContextProvider namespace={MODEL_REGISTRY_DEFAULT_NAMESPACE}>
+  <ModelRegistryContextProvider modelRegistryName="modelregistry-sample">
```

Comment:
I'm confused -- why would you pass in this value 🤔 Shouldn't the dropdown take care of this in the long run? Ie it's not a static or (in the future) a provided variable -- it's decided by the nature of the dropdown's selection

Reply:
Yes, that's rigth, that's the logic I'm working on in the follow up pr, in which the dropdown will have a modelregistry selector and will pass the name of that model registry to the context provider, that's why I left the TODO there